.gitlab/custom-jobs-and-variables.yml

```diff
@@ -68,5 +68,6 @@ variables:
 # CI behavior.
 # See Umpire for an example (export junit test reports).
 .custom_job:
-  variables:
-    JOB_TEMPLATE_CANNOT_BE_EMPTY: "True"
```

Comment:
@adrienbernede do you think I need to keep this variable?

Reply:
That's up to the project. Keep it as a reminder, or remove it because it's unlikely you go back to an empty job. (I remove it when that's up to me).